src/mainwindow.cpp

```diff
@@ -3071,7 +3071,7 @@ void MainWindow::ignoreMissingData_clicked()
   } else {
     ModInfo::Ptr info = ModInfo::getByIndex(m_ContextRow);
     //QDir(info->absolutePath()).mkdir("textures");
-    info->testValid();
+    info->diskContentModified();
```

Comment:
I tested removing this locally and it seemed to work fine so you can remove these as well as the commented out line above.